src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -1,5 +1,19 @@
 <template>
   <div id="dissolution-firm-form">
+    <v-card
+      outlined class="message-box rounded-0"
+    >
+      <p>
+        <strong>Important:</strong> You are about to dissolve
+        <b class="capitalize">{{ this.getBusinessLegalName }}</b>.
+        Once this process is completed and the required documents are
+        filed, the {{ this.corpTypeDescription() }} will
+        be struck from the register and dissolved, ceasing to be a registered
+        business under the Partnership Act.
+        If you make changes to your address below, please update your address in the account
```

Comment:
Do you intend to have a newline here? Because this is a paragraph element, it will be one long paragraph. Use a new paragraph element if needed.